### PR TITLE
RATEST-174: Internal patient transfer test workflow

### PR DESCRIPTION
**Pull request by Irenyak1**
Ticket [RATEST-174](https://issues.openmrs.org/browse/RATEST-174)

Fixed  -> Internal Patient Transfer Test Workflow

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "refappSelenium": "mvn test -Dcucumber.filter.tags='@selenium and not @initialSetup'",
     "refapp2VitalsAndTriagingTest": "mvn test -Dcucumber.filter.tags='@selenium and @vitals'",
     "refapp2ClinicalVisitTest": "mvn test -Dcucumber.filter.tags='@selenium and @clinicalVisit'",
+ 	"refapp2InternalPatientTransferTest":"mvn test -Dcucumber.filter.tags='@selenium and @internalTransfer'",
```
Comment (sherrif10) [inline package.json:14]: Put this on the same line.Thanks.

File: src/test/java/org/openmrs/contrib/qaframework/automation/InternalPatientTransferSteps.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+		admitToInpatientPage.confirm("Inpatient Ward");
+		admitToInpatientPage.waitForPage();
+	}
+
+	@Then("the system adds the patient data to admission table")
+	public void systemAddsPatientData() {
+		assertTrue(textExists("Encounters"));
+	}
+
+	@When("a user clicks on Transfer To Ward Service link from Patient dashboard")
```
Comment (sherrif10) [inline src/test/java/org/openmrs/contrib/qaframework/automation/InternalPatientTransferSteps.java:63]: You can use @And for other scenarios, try to minimize using @when for better workflow.

Comment (Irenyak1) [reply src/test/java/org/openmrs/contrib/qaframework/automation/InternalPatientTransferSteps.java:63]: ok

**Comment by sherrif10** [comment]
Well done @Irenyak1 , this looks amazing progress. Few comments left to get things merged.However cc @kdaud might have some suggestions.

**Comment by kdaud** [comment]
@sherrif10 see my comment have attached on the ticket comment section !

File: package.json (hunk 1 above)
Comment (Irenyak1) [reply package.json]: I have tried this like 8 times and I don't seem to get the exact spacing. On my machines it shows it's well aligned but here it's a different case. I there some sort of linter or formatter that you are using?

**Comment by kdaud** [comment]
See the comment on the [ticket](https://issues.openmrs.org/browse/RATEST-174)

**Pull request closed without merging**